### ISSUE TITLE
Can't start these maps (SoD, AB, RoE) 

### ISSUE THREAD

**Issue by GrafDeVan** [vcmi/vcmi#2166]
Basically, all maps that didn't run under SoD run in HotA. Here they are [Looks like maps from SoD but launched only in HOTA.zip](https://github.com/user-attachments/files/28949809/Looks.like.maps.from.SoD.but.launched.only.in.HOTA.zip) Even though they're marketed as SoD maps. I don't know what's wrong; maybe it's some WoG objects, HotA objects, or something else.

Only this SoD map won't start in Hota [SoD_Corbus_Realm_2.zip](https://github.com/user-attachments/files/28949964/SoD_Corbus_Realm_2.zip) Game just closes

> Old description
> 
> Hi. Some of these maps were launched a long time ago (SoDTheLastCrusade), some of them were never launched (C-_Shadows_of_the_Forest).
> The rest don't really work. And I don't know their behavior.
> [Maps.zip](https://github.com/vcmi/vcmi/files/11504939/Maps.zip)

**Comment by Povelitelh3** [comment]
version is current develop 1.3.0.7a54273b7c22147b8898ed87d634a2f89387c09b mingw-ubuntu

**Comment by IvanSavenko** [comment]
Checked all provided maps:
- Incorrect handling of Troll in army. Will be fixed.
- Heroes have war machines in the army. Will be removed on load.
- Heroes have "Ironfist of the Ogre" - unfinished artifact from Heroes 3. Will be removed on load.
- Instant loss since hero specified in loss condition exists only as campaign placeholder. H3 replaces loss condition with generic one.
- Instant loss since hero specified in loss condition is in prison. H3 applies condition only once imprisoned hero is freed. 
- Instant loss since AI owns target town on day 1 and victory condition applies to AI. H3 ignores AI who already owns town?

However, some maps are unlikely to be fixed. Specifically:
- Attainment - File is corrupted
- Battle - Contains WoG objects
- Crisis of Destiny - Contains WoG objects
- Shadows of the Forest - Object outside map bounds. Not supported at the moment.
- Rise of the Sun King - Game hangs on start in VCMI due to unknown reason. Crashes in Heroes 3.

**Comment by rilian-la-te** [comment]
@IvanSavenko we should have an ability to replace OH3's Ironfist of the Ogre with Hota one, at least in editor.

**Comment by Povelitelh3** [comment]
@IvanSavenko Added map which doesn't work in 1.6.3

[The Legend of Fairy Sword IV (CHI).zip](https://github.com/user-attachments/files/18468232/The.Legend.of.Fairy.Sword.IV.CHI.zip)
[AB_A_War.zip](https://github.com/user-attachments/files/18468285/AB_A_War.zip)
[AB_Angor_s_Wrath.zip](https://github.com/user-attachments/files/18468290/AB_Angor_s_Wrath.zip)
[Atomium1.2.zip](https://github.com/user-attachments/files/18468287/Atomium1.2.zip)
[Defend Taierzhuang (CHI).zip](https://github.com/user-attachments/files/18468293/Defend.Taierzhuang.CHI.zip)
[Doomsday Battle-BT (CHI).zip](https://github.com/user-attachments/files/18468288/Doomsday.Battle-BT.CHI.zip)
[Dragon's Nest (CHI).zip](https://github.com/user-attachments/files/18468292/Dragon.s.Nest.CHI.zip)
[Jedi Strikes Back-BT (CHI).zip](https://github.com/user-attachments/files/18468291/Jedi.Strikes.Back-BT.CHI.zip)
[Saint Seiya the Goddess of Wisdom-BT (CHI).zip](https://github.com/user-attachments/files/18468289/Saint.Seiya.the.Goddess.of.Wisdom-BT.CHI.zip)
[SoD__Школьные_Войны_.zip](https://github.com/user-attachments/files/18468295/SoD__._._.zip)
[SoD_Dart_NeoEdik__часть_2.zip](https://github.com/user-attachments/files/18468294/SoD_Dart_NeoEdik__._2.zip)
[SoD_Сага_о_Мече_.zip](https://github.com/user-attachments/files/18468286/SoD_._._._.zip)

**Comment by Povelitelh3** [comment]
And these are the Hota maps. They also all don't start. I don't know how to view their version. At least there are no errors in the console when you enter a new game

![Image](https://github.com/user-attachments/assets/8b840c49-010d-487a-b4d2-dd4ecc41e16c)

[aeten_nomere_.zip](https://github.com/user-attachments/files/18468323/aeten_nomere_.zip)
[carnival_abyss_1964.zip](https://github.com/user-attachments/files/18468322/carnival_abyss_1964.zip)
[conquest_of_the_universe_1873.zip](https://github.com/user-attachments/files/18468321/conquest_of_the_universe_1873.zip)
[the_bogmire_brothers_1490.zip](https://github.com/user-attachments/files/18468324/the_bogmire_brothers_1490.zip)

**Comment by GeorgeK1ng** [comment]
Tested now aeten nomere map. Can be loaded in Map Editor fine, but can't play it.

`Open map file: MAPS/[HotA] [EN] AetenNomere 1.9
Template animation AVMgeWL1 of type (53 5) is missing!
Map '[hota] [en] aetennomere 19': Wandering monsters (76 18 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (140 112 0) - 5 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (139 7 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (98 102 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (22 79 0) - 5 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (107 141 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (1 98 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (64 131 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (87 111 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (102 40 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (142 53 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (143 53 0) - 6 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (94 99 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (42 64 0) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (22 17 0) - 5 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (89 109 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (50 80 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (87 107 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (115 94 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19: creature bank at (69 57 0) settings 3 -1 0 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (75 64 0) settings -1 0 100 -1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (3 18 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (48 76 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (91 6 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (129 29 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (120 105 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (84 6 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (136 85 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (16 48 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (91 63 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (87 64 0) settings -1 0 100 -1 1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (90 75 0) - 5 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (80 81 0) settings -1 0 100 -1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (64 83 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (114 51 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (9 35 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (19 49 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (91 24 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (83 95 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (37 130 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (11 133 0) - 4 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (88 8 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (3 45 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (44 137 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (41 67 0) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (16 55 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (108 70 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (16 40 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (58 15 0) - 6 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (94 21 0) settings -1 0 75 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (82 104 0) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (4 135 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (10 98 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19: creature bank at (105 101 0) settings 3 -1 1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (94 113 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (113 109 0) - 6 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (51 69 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (4 102 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (144 44 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19: creature bank at (9 38 0) settings 3 -1 0 are not implemented!
Map '[hota] [en] aetennomere 19: creature bank at (38 54 0) settings 3 -1 1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (75 114 0) - 6 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (75 115 0) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19: creature bank at (31 134 0) settings 3 1 0 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (102 10 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (55 54 0) - 6 quests are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (55 54 0) - 1 repeatable quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (14 103 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (123 7 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (120 74 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (52 98 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (27 47 0) settings -1 0 100 -1 3 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (133 75 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (89 88 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (69 118 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (85 16 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (87 46 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (12 106 0) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (75 104 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (46 68 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (39 88 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (79 23 0) - 5 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (79 24 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (56 71 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (47 72 0) settings -1 0 100 0 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (104 102 0) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (18 54 0) settings -1 0 100 -1 1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (84 52 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (109 143 0) - 5 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (108 114 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (112 117 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (4 69 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (62 9 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (11 49 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (13 43 0) settings -1 1 50 1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (15 87 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (17 93 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (30 107 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (10 95 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (105 75 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (20 89 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (73 121 0) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (88 120 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (108 11 0) - 6 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (108 12 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (85 26 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (138 48 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (9 41 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (44 131 0) settings -1 0 100 -1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (124 92 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (70 105 0) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (116 95 0) settings -1 0 50 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (84 128 0) settings -1 0 100 1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (143 83 0) settings -1 1 25 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (118 84 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (123 72 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (140 76 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (124 107 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (127 111 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (132 108 0) settings -1 0 100 -1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (98 89 0) settings -1 0 100 1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (51 29 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (107 88 0) settings -1 0 100 -1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (5 90 0) settings -1 0 100 -1 1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (31 55 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (31 40 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (99 37 0) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (93 79 0) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (61 139 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (58 137 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (83 20 0) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (89 50 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (6 110 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (17 110 0) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (86 39 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (43 43 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (46 51 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (52 42 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (51 46 0) settings -1 0 100 -1 -2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (124 102 0) settings -1 0 100 -1 3 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (84 69 0) settings -1 0 100 -1 3 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (75 69 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (107 111 0) settings -1 0 75 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (74 52 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (42 31 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (83 118 0) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (82 143 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (41 45 0) settings -1 0 100 1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (23 79 0) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (75 89 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (56 87 0) - 7 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (87 3 0) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (97 92 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (121 100 0) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (28 99 0) settings -1 0 50 -1 1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (23 17 0) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (43 59 0) settings -1 0 100 -1 5 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (9 56 0) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (39 121 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (10 130 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (22 140 1) settings -1 1 100 1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (79 135 1) - 7 quests are not implemented!
Map '[hota] [en] aetennomere 19: creature bank at (105 9 1) settings 1 -1 0 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (105 10 1) settings -1 0 100 -1 1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (97 4 1) settings -1 0 50 1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (112 8 1) settings -1 0 100 1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (112 139 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (142 117 1) settings -1 0 100 -1 1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (92 92 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (85 74 1) settings -1 0 100 -1 1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (121 67 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (75 9 1) settings -1 0 100 -1 1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (118 39 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (126 116 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (119 17 1) settings -1 0 75 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (142 50 1) settings -1 0 100 1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (136 78 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (139 71 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (142 61 1) settings -1 0 50 1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (126 104 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (116 97 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (104 106 1) settings -1 0 75 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (74 104 1) settings -1 0 75 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (36 133 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (25 100 1) - 9 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (24 100 1) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (83 77 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (99 132 1) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (72 110 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (55 122 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (49 123 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (130 14 1) settings -1 0 75 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (130 62 1) settings -1 0 50 1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (91 104 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (71 58 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (128 74 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (61 132 1) settings -1 0 100 0 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (50 131 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (45 27 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (139 103 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (119 112 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (134 127 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (56 137 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (119 133 1) settings -1 0 100 0 5 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (125 127 1) settings -1 0 100 1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (143 121 1) settings -1 0 75 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (93 81 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (116 84 1) - 6 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (117 80 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (115 85 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (109 95 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (91 100 1) settings -1 0 75 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (142 28 1) - 4 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (142 29 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (124 8 1) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (13 121 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (63 87 1) settings -1 0 100 1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (68 82 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (84 107 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (60 120 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (84 31 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (86 25 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (78 14 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19: creature bank at (108 112 1) settings 3 -1 0 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (108 113 1) settings -1 0 100 -1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (98 110 1) settings -1 0 25 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (106 104 1) settings -1 0 25 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (142 79 1) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (66 127 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (37 93 1) settings -1 1 50 1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (108 53 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (110 59 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (103 54 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (101 61 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (52 7 1) settings -1 0 100 -1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (59 97 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (50 67 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (58 67 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (113 36 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (76 36 1) settings -1 0 100 -1 4 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (39 47 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (43 48 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (40 56 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (33 35 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (55 53 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (60 9 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (8 76 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (24 128 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (83 98 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (77 96 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (101 77 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (85 87 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (120 14 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19: creature bank at (71 36 1) settings 0 -1 0 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (66 39 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (71 37 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (64 30 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (61 18 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (41 93 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (44 82 1) settings -1 0 100 -1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (87 14 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (104 30 1) settings -1 1 100 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (99 27 1) settings -1 0 50 1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (5 98 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (143 5 1) settings -1 0 100 -1 3 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (136 3 1) settings -1 0 100 1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (120 4 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (141 45 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (110 88 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (88 28 1) - 8 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (87 29 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (104 139 1) - 4 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (103 140 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (14 2 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (144 133 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (32 2 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (2 22 1) settings -1 0 75 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (4 77 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (11 10 1) settings -1 0 50 -1 4 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (10 15 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (141 54 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (33 67 1) settings -1 0 100 0 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (44 72 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (33 75 1) settings -1 0 75 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (54 70 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (5 58 1) - 4 quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (6 59 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (25 42 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (120 57 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (114 51 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (119 53 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (125 54 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (10 75 1) settings -1 1 25 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (9 117 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (95 48 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (4 11 1) settings -1 0 25 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (44 36 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (41 29 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (99 129 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (115 141 1) settings -1 0 50 1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (63 49 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (63 53 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (41 9 1) settings -1 0 100 1 6 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (30 70 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (53 20 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (37 59 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (2 3 1) settings -1 0 50 -1 5 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (43 7 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (35 46 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (40 52 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (44 54 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (34 113 1) - 1 repeatable quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (35 114 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (82 62 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (77 71 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (67 85 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (86 66 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (22 2 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (31 55 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (126 27 1) settings -1 0 100 -1 2 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (39 5 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (39 72 1) settings -1 0 50 1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (75 76 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (46 70 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (36 139 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (69 24 1) settings -1 1 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (16 125 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (57 98 1) settings -1 0 25 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (75 75 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (14 135 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (2 131 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (51 113 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (12 81 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (21 85 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (11 37 1) settings -1 1 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (15 35 1) settings -1 0 100 0 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (111 12 1) - 3 quests are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (111 12 1) - 1 repeatable quests are not implemented!
Map '[hota] [en] aetennomere 19': Wandering monsters (111 13 1) settings -1 0 50 -1 -1 are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (11 36 1) - 5 quests are not implemented!
Map '[hota] [en] aetennomere 19': Seer Hut at (11 36 1) - 1 repeatable quests are not implemented!
Map loaded!
Object 79:3 at (45 67 0) has no templates suitable for terrain Voda
Hero Christian already has artifact at 13, omitting giving artifact 4
Hero Gem already has artifact at 15, omitting giving artifact 6
Hero Torosar already has artifact at 13, omitting giving artifact 4
Hero Pyre already has artifact at 13, omitting giving artifact 4
Hero Gurnisson already has artifact at 13, omitting giving artifact 4
Invalid syntax in load value! Can not load value from 'navigation'
Invalid syntax in load value! Can not load value from 'navigation'
Invalid syntax in load value! Can not load value from 'navigation'
Invalid syntax in load value! Can not load value from 'navigation'
Invalid syntax in load value! Can not load value from 'navigation'
Gamestate initialized!
Failed to find object of type 63::51
Disaster happened.
Reason: 0xe06d7363 - UNKNOWN EXCEPTION at 0033:0,000,7FF,A5F,1EC,B69
Thread ID: 20292
Crash info will be put in VCMI_client.exe_crashinfo.dmp`

**Comment by Olegmods** [comment]
> And these are the Hota maps. They also all don't start. I don't know how to view their version. At least there are no errors in the console when you enter a new game
> 
> ![Image](https://github.com/user-attachments/assets/8b840c49-010d-487a-b4d2-dd4ecc41e16c)
> 
> [aeten_nomere_.zip](https://github.com/user-attachments/files/18468323/aeten_nomere_.zip) [carnival_abyss_1964.zip](https://github.com/user-attachments/files/18468322/carnival_abyss_1964.zip) [conquest_of_the_universe_1873.zip](https://github.com/user-attachments/files/18468321/conquest_of_the_universe_1873.zip) [the_bogmire_brothers_1490.zip](https://github.com/user-attachments/files/18468324/the_bogmire_brothers_1490.zip)

Googling the first map said that it's 1.7.1 map.
Probably the same with the other ones.

You can check if they are 1.7 or not by using 1.6 HotA editor. If these won't work there, same applies to VCMI.

**Comment by Povelitelh3** [comment]
> Googling the first map said that it's 1.7.1 map.

Googling every map is definitely not for me. Moreover, maps can be updated from version to version, in fact, it can be from an older one, and googling a newer one.
But the second option suits me. For some reason, I did not think about it

**Comment by IvanSavenko** [comment]
- HotA 1.7 maps should not show up in the map list in game.
- HotA 1.6 maps should always be visible if you have hota mod (and generally should start, not counting bugs)

So no need to search for that map / open it in editor

**Comment by GrafDeVan** [comment]
It looks like the same old problems again. It is impossible to start a game on these maps. Moreover, it was possible to play on some of them before, for example on maps like: Xiedu,  Paragon, 2 Friends etc

SoD [MapPackforDevelopers.zip](https://github.com/user-attachments/files/22366927/MapPackforDevelopers.zip)

msvs x64

**Comment by GrafDeVan** [comment]
@MichalZr6 The issue has only been partially fixed. These maps still cause crashes.

[MapPackforDevelopers 2.zip](https://github.com/user-attachments/files/22708795/MapPackforDevelopers.2.zip)

msvs x64, 1.7.0.4fcca634eab618b49e62745646bd62439f827de5

**Comment by GrafDeVan** [comment]
this Hota map also doesn't work

[Into the Depths of Hell.zip](https://github.com/user-attachments/files/22709048/Into.the.Depths.of.Hell.zip)

**Comment by MichalZr6** [comment]
Thanks, I will test with these you provided.

**Comment by MichalZr6** [comment]
> this Hota map also doesn't work
> 
> [Into the Depths of Hell.zip](https://github.com/user-attachments/files/22709048/Into.the.Depths.of.Hell.zip)

This one is another story.
The random town on this map is placed like this:

<img width="698" height="766" alt="Image" src="https://github.com/user-attachments/assets/40df9662-d464-44f8-842e-a68d9b395549" />

You can see only its right flag.
I patched the code so the map becomes playable, opening PR:
- #6218

**Comment by MichalZr6** [comment]
"A War" map from `MapPackforDevelopers 2.zip` has AB header but it was probably edited in Hota editor and contains RiverId == 9 which is incorrect for AB maps (RiverId <= features.riversCount = 4).
This PR should allow to run this (and probably some other) map:
- #6221

**Comment by GrafDeVan** [comment]
>  This PR should allow to run this (and probably some other) map:

>     * [Fix buffered stream growth & decode river type from low 3 bits (compat for “A War” AB map) #6221](https://github.com/vcmi/vcmi/pull/6221)


Good Job. Keep work. Map "A War" works fine now

**Comment by MichalZr6** [comment]
The AB map - “Angor’s Wrath” - has Hero Placeholder (why ?) at tile = (2, 33) which is Sir Mullich. It crashed the client in VCMI so I tweaked the code and it runs, but instantly ends as having this hero is the loss condition. Funny thing is that Hota runs this map, but without that hero. Any thoughts how this case should be handled ?

**Comment by GeorgeK1ng** [comment]
Some kind of ERM script / timed event possible for this case?

**Comment by MichalZr6** [comment]
Possible. I didn't have enough time to test how it goes further on in Hota. @GrafDeVan do you know this map ?

**Comment by GrafDeVan** [comment]
> Possible. I didn't have enough time to test how it goes further on in Hota. [@GrafDeVan](https://github.com/GrafDeVan) do you know this map ?

What should I know about her? I can't figure it out. Of course I do, I added her to this pack :)
Which PR fixes this map?

**Comment by GeorgeK1ng** [comment]
> > Possible. I didn't have enough time to test how it goes further on in Hota. [@GrafDeVan](https://github.com/GrafDeVan) do you know this map ?
> 
> What should I know about her? I can't figure it out. Of course I do, I added her to this pack :) Which PR fixes this map?

Something from gameplay outside VCMI engine to figure out hidden Sir Mullich I quess?

**Comment by GrafDeVan** [comment]
All maps are worked but in Hota except one. description was updated. almost done, may be need to close.
